diff --git a/zebra-crosslink/zebrad/src/commands/start.rs b/zebra-crosslink/zebrad/src/commands/start.rs
index d1aa91d..3ce887b 100644
--- a/zebra-crosslink/zebrad/src/commands/start.rs
+++ b/zebra-crosslink/zebrad/src/commands/start.rs
@@ -676,7 +676,7 @@ impl StartCmd {
                     cookie_dir: None,
                 }),
                 grpc_settings: zaino_serve::server::config::GrpcServerConfig {
-                    listen_address: std::net::SocketAddr::new(std::net::IpAddr::V4(std::net::Ipv4Addr::new(0, 0, 0, 0)), zebra_port_base + 10001),
+                    listen_address: std::net::SocketAddr::new(std::net::IpAddr::V4(std::net::Ipv4Addr::new(127, 0, 0, 1)), zebra_port_base + 10001),
                     tls: None,
                 },
                 validator_settings: zaino_common::ValidatorConfig {
